apps/sms/services/js/device_storage/device_storage_client.js

```diff
@@ -0,0 +1,70 @@
+/* global bridge,
+          BroadcastChannel */
+
+(function(exports) {
+  'use strict';
+
+  /**
+   * Name of the service that is responsible for device storage functionality.
+   * @type {string}
+   */
+  const SERVICE_NAME = 'device-storage-shim';
+
+  // TODO: Remove the flag once the low storage feature is completed.
+  const LOW_STORAGE_ENABLE = false;
```

Comment:
__nit:__ agree with Julien that it would be better to keep this in Settings (if possible).